tox.ini

```diff
@@ -7,7 +7,7 @@ deps=
     -rrequirements/compiled.txt
     -rrequirements/dev.txt
 commands =
-    coverage run manage.py test --noinput -v2 kuma
+    coverage run manage.py test --noinput -v2 {posargs:kuma}
```

Comment:
This change is also in https://github.com/mozilla/kuma/pull/3109/files#diff-b91f3d5bd63fcd17221b267e851608e8 so it may potentially conflict?